ansible/playbooks/roles/video-stream-dump/tasks/configure_streamdump.yml

```diff
@@ -25,31 +26,32 @@
     group: root
     mode: 0644
   with_items: "{{ video_dump_source }}"
-  register: streamdump
+  register: video_stream_dump_services
+  notify:
+  - restart streamdump service
 
-- name: perform a daemon-reload if any of the stream dump services have been modified
+- name: enable streamdump service
   systemd:
-    daemon_reload: yes
-  when: streamdump.changed
-
-- name: enable stream dump services
-  service:
     name: "streamdump-{{ item.host }}"
     enabled: yes
     state: started
+    daemon_reload: yes
   with_items: "{{ video_dump_source }}"
 
-- name: restart service if video-network-monitor service or script has been modified
-  service:
-    name: "streamdump-{{ item.host }}"
-    state: restarted
-  with_items: "{{ video_dump_source }}"
-  when: streamdump.changed or streamdumpscript.changed
+- name: install streamdump target
+  template:
+    src: streamdump.target
+    dest: /etc/systemd/system/streamdump.target
+    owner: root
+    group: root
+    mode: 0644
+  notifies:
```

Comment:
Does this work? Shouldn't it be `notify`?

Reply:
Whups, typo.